docs/user-guide/install-zowe-smpe.md

```diff
@@ -764,6 +774,9 @@ __Expected Return Codes and Messages:__ You will receive a return code of 0 if t
 
 Edit and submit sample job ZWE3ALOC to allocate the SMP/E target and distribution libraries for Zowe. Consult the instructions in the sample job for more information.
 
+<!--Issue: DLIB and TARGET do not default - maybe provide sensible defaults
+Suggestion: You would hope that the installer knows SMP/E, but adding a suggestion of ZWET0 and ZWED0 for the target and distribution libraries does no harm.-->
```

Comment:
@NayerNajafi @Joe-Winchester Do we have any suggestion of ZWET0 and ZWED0 for the target and distribution libraries?

Reply:
@nannanli Ashley, 
1) If this issue is related to **the default value for DLIB and TARGET zones (ZWE1SMPE jcl)**, as far as I know, z/OS products don't have default value for Target and Distribution zones while using SMP/E installation. System programmer will define that.

```
3) Change #tzone to your CSI target zone name.
4) Change #dzone to your CSI distribution zone name.

```

2) If the issue is about **HLQ for DLIB and TARGET libraries (ZWE3ALOC jcl)**,  z/OS products don't provide the default HLQ. 
In the JCL, we ask z/OS sysprog to define the HLQ:
```
2) Change #thlq to the appropriate high level qualifier(s) for    
   the target data sets. The maximum length is 35 characters.     
                                                                  
3) Change #dhlq to the appropriate high level qualifier(s) for    
   the distribution datasets. The maximum length is 35 characters.
```
But my answer for this issue would be: 
Adding default value for the target and distribution libraries/zones does not cause any issues, but it might not meet basic standards of IBM SMP/E packaging and installation.